readme.md

```diff
@@ -658,6 +658,7 @@
 - [TypeDB](https://github.com/vaticle/typedb-awesome#readme) - Logical database to organize large and complex networks of data as one body of knowledge.
 - [Cassandra](https://github.com/Anant/awesome-cassandra#readme) - Open-source, distributed, wide column store, NoSQL database management system.
 - [TDengine](https://github.com/taosdata/awesome-tdengine#readme) - An open-source time-series database with high-performance, scalability, and SQL support.
+- [Supabase](https://github.com/lyqht/awesome-supabase) - An open source alternative to Firebase that provides a Postgres database, Authentication, instant APIs, Edge Functions, Realtime subscriptions, and Storage.
```

Comment:
The description is too long and should not be title-cased.

Reply:
have revised this!